src/main/java/org/reaktivity/nukleus/kafka/internal/stream/NetworkConnectionPool.java

```diff
@@ -1705,8 +1710,8 @@ private void handleMetadataResponse(
             {
                 final TopicMetadataFW topicMetadata =
                         NetworkConnectionPool.this.topicMetadataRO.wrap(networkBuffer, networkOffset, networkLimit);
-                final String16FW topicName = topicMetadata.topic();
-                assert topicName.asString().equals(pendingTopicMetadata.topicName);
+                String16FW topicNameFW = topicMetadata.topic();
```

Comment:
We typically don't use `FW` suffix, recommend to keep as `topicName`.

Reply:
done